resqpy/time_series/time_series.py

```diff
@@ -0,0 +1,158 @@
+"""time_series.py: RESQML time series class."""
+
+version = '16th September 2021'
+
+# Nexus is a registered trademark of the Halliburton Company
```

Comment:
This comment should be included in files, and only those files, where 'Nexus' appears somewhere.